rotkehlchen/chain/evm/decoding/thegraph/decoder.py

```diff
@@ -173,3 +222,10 @@ def counterparties() -> tuple[CounterpartyDetails, ...]:
 
     def addresses_to_decoders(self) -> dict[ChecksumEvmAddress, tuple[Any, ...]]:
         return {self.staking_contract: (self._decode_delegator_staking,)}
+
+    def decoding_by_input_data(self) -> dict[bytes, dict[bytes, Callable]]:
+        return {
+            DEPOSIT_ETH: {
+                ETH_DEPOSITED: self._decode_contract_deposit,
+            },
+        }
```

Comment:
In the transaction it seems that you interact with https://etherscan.io/address/0xca82c7ce3388b0b5d307574099ac57d7a00d509f and seems to be the case for all the users. IF you could investigate what that contract is and if it is the same one for everyone that would allow to remove this logic and use normal decoding by event log address. Please ask to confirm if that is the case